.copier-answers.yml

```diff
@@ -0,0 +1,14 @@
+# Changes here will be overwritten by Copier
+_commit: 1.0.0
+_src_path: gh:DiamondLightSource/python-copier-template
+author_email: tom.cobb@diamond.ac.uk
+author_name: Tom Cobb
+component_owner: user:default/tmc43
```

Comment:
Please can you change the `component_owner` as an answer rather than in the generated files please?